build-farm/platform-specific-configurations/linux.sh

```diff
@@ -177,25 +177,38 @@ elif [ -r /usr/bin/gcc-7 ]; then
   [ -r /usr/bin/g++-7 ] && export CXX=/usr/bin/g++-7
 fi
 
-# Bisheng on aarch64 has a KAE option which requires openssl 1.1.1 to be used
-if [ "${VARIANT}" == "${BUILD_VARIANT_BISHENG}" ] && [ -x /usr/local/openssl-1.1.1/lib/libcrypto.so.1.1 ]; then
-  export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-extra-cflags=-I/usr/local/openssl-1.1.1/include  --with-extra-cxxflags=-I/usr/local/openssl-1.1.1/include --with-extra-ldflags=-L/usr/local/openssl-1.1.1/lib"
+if [ "${VARIANT}" == "${BUILD_VARIANT_BISHENG}" ]; then
+  # BUILD_C/CXX required for native (non-cross) RISC-V builds of Bisheng
+  if [ ! -z "$CXX" ]; then
+    export BUILD_CC="$CC"
+    export BUILD_CXX="$CXX"
+  fi
+  # Bisheng on aarch64 has a KAE option which requires openssl 1.1.1 to be used
+  if [ -x /usr/local/openssl-1.1.1/lib/libcrypto.so.1.1 ]; then
```

Comment:
Worth parameterizing this variable/number, I see its repeated below.

Reply:
Below as in the next line? OK I will change that, but I personally find that when it's only used locally it's a LOT easier to be able to cut & paste from the script when debugging instead of having to manually dereference variables.